.azure-pipelines/model-test.yml

```diff
@@ -0,0 +1,120 @@
+trigger: none
+
+pr:
+  autoCancel: true
+  drafts: false
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - onnx_neural_compressor
+      - setup.py
+      - requirements.txt
+      - .azure-pipelines/scripts/models
+      - .azure-pipelines/model-test.yml
+      - .azure-pipelines/template/model-template.yml
+    exclude:
+      - test
+
+variables:
+  OUT_SCRIPT_PATH: $(Build.SourcesDirectory)/.azure-pipelines/scripts/models
+  SCRIPT_PATH: /neural_compressor/.azure-pipelines/scripts
+
+parameters:
+  - name: algorithms
+    type: object
+    default:
+      - SQ
+      - WOQ
+  - name: models
+    type: object
+    default:
+      - bert_base_MRPC
+      - bert_base_MRPC_dynamic
+      - resnet50-v1-12_qdq
+      - resnet50-v1-12
+
+stages:
+  # - stage: ONNX_LLM_Models
```

Comment:
Any issues for this example, I think it's better to add SQ and WOQ examples in CI.